awx/ui_next/src/components/FormLayout/FormLayout.jsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+export const FormColumnLayout = styled.div`
+  width: 100%;
+  grid-column: 1 / 4;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+
+  @media (min-width: 1210px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+`;
+
+export const FormFullWidthLayout = styled.div`
+  grid-column: 1 / -1;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 20px;
+`;
+
+export const FormCheckboxLayout = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+
+  & > * {
+    margin-right: 30px;
+    margin-bottom: 10px;
+  }
+`;
+
+export const SubFormLayout = styled.div`
+  grid-column: 1 / -1;
+  background-color: #f5f5f5;
+  margin: 0 -24px;
```

Comment:
This -24px is tied to the fact that our forms are inside of Cards, but it's possible that won't always be the case (esp. in modals).

This isn't 100% fullproof, but one option is to delete this line and replace with:

```
margin-right: calc(var(--pf-c-card--child--PaddingRight) * -1);
margin-left: calc(var(--pf-c-card--child--PaddingLeft) * -1);
```

I don't think these custom properties are defined outside a Card